lib/VOMSLibrary.py

```diff
@@ -43,6 +44,30 @@ def compare_dates(self, date0, date1):
 		else:
 			return 1
 
+	def parse_vomses_files(self, vomses_dirs: list = ["/etc/vomses", "~/.glite/vomses"]) -> dict:
+		"""Read the files in /etc/vomses and in ~/.glite/vomses, returning a dictionary with the following items:
+		vo: [hosts]"""
+
+		vomses_files = []
+		for vomses_dir in vomses_dirs:
+			if os.path.exists(os.path.expanduser(vomses_dir)):
```

Comment:
I would save `os.path.expanduser(vomses_dir)` in a var and reuse it for the join in the next line